packages/adapter-onebot/package.json

```diff
@@ -31,7 +31,9 @@
   },
   "devDependencies": {
     "@types/ws": "^7.4.2",
+    "chai": "^4.3.4",
```

Comment:
chai is already in workspace root devDependencies, why do we need it here?

Reply:
hhhh, u1s1 qs

Reply:
Yarn 2 requires all dependencies of the current package to be listed in their own `package.json`, so that Yarn would know which dependency the current package needs and feed it.
In other words, if `chai` or something else is not needed on the workspace root, it should be removed from the root dependencies.

See also: https://yarnpkg.com/features/pnp#fixing-node_modules

Reply:
实际上我们并没有使用 yarn 2，另外使用起来也有很多额外的问题，并不单单解决了这一个就行。
我们之前也有讨论过给 koishi 换上 yarn 2 ，也实践过，会带来更多新的问题。
如果你是为了使用 yarn 2 我觉得需要开一个新的 pull request 。
In fact, we didn't use yarn 2, and there were a lot of additional problems when using it, and it didn't just solve this one.
We have also discussed replacing koishi with yarn 2 before, and we have also practiced it, which will bring more new problems.
If you are using yarn 2 I think you need to open a new pull request.

Reply:
Yeah, you're right.
But that's because @Shigma said in https://github.com/koishijs/koishi/pull/315#issuecomment-889119242 , so I add missing dependencies in every package that is imported.

To be honest, I think the migration to Yarn 2 is hard work because there are many cross-references and incompatibility from third-party tools, also `node_modules` path checking, et cetera.
This PR is just a minor tweak for users who are using the `PnP` feature to install koishi, make their Yarn 2 recognize dependencies correctly.

Reply:
emmmm, It makes sense